files/en-us/web/api/rtcicecandidate/relatedaddress/index.html

```diff
@@ -50,25 +41,21 @@ <h3 id="Value">Value</h3>
   base for that server or peer reflexive candidate. For relay candidates, the related
   address and port are set to the mapped address selected by the TURN server.</p>
 
-<p>For host candidates, <code>relatedAddress</code> is <code>null</code>, meaning the
-  field is not included in the candidate's a-line.</p>
+<p>For host candidates, <code>relatedAddress</code> is <code>null</code>, meaning the field is not included in the candidate's a-line.</p>
 
 <h2 id="Usage_notes">Usage notes</h2>
 
 <p>The related address is included in ICE candidates despite not being used by ICE itself.
   <code>relatedAddress</code> can be used for diagnostic purposes; by observing the
   relationships between the various types of candidates and their addresses and related
-  addresses. <code>relatedAddress</code> can also be used by Quality-of-Service (QoS)
-  mechanisms.</p>
+  addresses. <code>relatedAddress</code> can also be used by Quality-of-Service (QoS) mechanisms.</p>
 
-<p>Here's an {{Glossary("SDP")}} attribute line (a-line) describing an ICE candidate
-  discovered by the STUN server:</p>
+<p>Here's an {{Glossary("SDP")}} attribute line (a-line) describing an ICE candidate discovered by the STUN server:</p>
```

Comment:
While we are here, let's this red link.
```suggestion
<p>Here's an <a href="/en-US/docs/Web/API/WebRTC_API/Protocols#sdp">SDP</a> attribute line (a-line) describing an ICE candidate discovered by the STUN server:</p>
```